README.md

```diff
@@ -1,6 +1,14 @@
 # refinebio-processor-documentation
 For tracking documentation of the methodology behind our processor tests: https://github.com/AlexsLemonade/refinebio/issues/2470
 
+## New Required Tests
+
+In addition to improving the existing tests as listed in the links below, we also would like to run a few tests on a weekly basis to make sure things aren't drifting without us knowing.
+These can be configured to run as cron jobs on the foreman server.
+These should check:
+- that brainarray has not drastically changed genes since the last time it was run
+- that a newly generated QN reference file is similar enough to the latest version currently being used.
```

Comment:
@jaclyn-taroni should this be using Spearman correlation as well?

Reply:
I think we'd probably want to compare the distributions instead (Kolmogorov-Smirnov test is what immediately comes to mind). I'd say that this is fine for now and we can nail down specifics later.